python/lsst/ts/scriptqueue/script_queue.py

```diff
@@ -177,6 +186,30 @@ def _get_scripts_path(self, patharg, is_standard):
             raise ValueError(f"{category} scripts path {dir_path} is not a directory")
         return dir_path
 
+    def _make_temp_ospl_configuration(self):
```

Comment:
Is this going to interfere with the producer and everyone else from getting data from scripts?

Reply:
No. It will have no visible effect. But this complex code is not needed if Dave agrees to a simple proposal (see discussion in Jira)